notebooks/04_model_select_and_optimize/aml_hyperparameter_tuning.ipynb

```diff
@@ -0,0 +1,619 @@
+{
```

Comment:
<p>One question here, is this agnostic to the model? Meaning, can I use any model (TF, LightGBM, sklearn) with this notebook?</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Good question (and good point to discuss further). You should change mostly the first part and last part of the notebook since each model will have different parameters. Also will need to change the parsing script (AzureML's entry script).</p><p><br></p><p>But would be nice to generalize more...</p>